text/0864-deprecate-ember-a.md

```diff
@@ -0,0 +1,94 @@
+---
+stage: accepted
+start-date: 2022-11-22T00:00:00.000Z
+release-date:
+release-versions:
+teams:
+  - data
+  - framework
+  - learning
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/864
+project-link:
+---
+
+# Deprecate Ember.A()
+
+## Summary
+
+Remove `Ember.A()` as the functionality it provides is no longer needed in the post octane reactivity system and with the removal of array prototype extensions in #848.
```

Comment:
I think we also may should deprecate `Ember.NativeArray` as part of this. I'd also like to deprecate `Ember.Array` altogether, but I'm not sure if that should be a separate thing or not. At the very least it would require the work we're doing here.